src/swift/Sources/main.swift

```diff
@@ -41,5 +41,5 @@ SharedLogger.debug("Got to the end of main. Exiting")
 
 func exit() {
```

Comment:
This function is going to be deleted. I used it to pause execution way back when there was no output.